drools-reliability/drools-reliability-h2mvstore/pom.xml

```
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.drools</groupId>
     <artifactId>drools-reliability</artifactId>
-    <version>8.41.0-SNAPSHOT</version>
+    <version>8.42.0-SNAPSHOT</version>
```

```suggestion
    <version>${project.version}</version>
```

Should avoid this problem in the future?

---

@tarilabs I think `${project.version}` is not possible when referring `parent` because it's the root information, right?

---

doh sorry you are absolutely right, i got misled by the github UI I thought this was a dependency! thanks for clarifying.